lib/rubocop/cop/rspec/builtin_matcher.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Prefer to use built-in matcher.
+      #
+      # @example
+      #   # bad
+      #   expect(foo).to be_exist(bar)
+      #   expect(foo).not_to be_include(bar)
+      #
+      #   # good
+      #   expect(foo).to exist(bar)
+      #   expect(foo).not_to include(bar)
+      #
+      class BuiltinMatcher < Base
```

Comment:
There’s a wip cop naming guideline saying yhat the name should indicate the offence. I’d suggest RedundantPredicateMatcherUse

Reply:
I see. How about `RedundantPredicateMatcher`?